doc/hardware/peripherals/charger.rst

```diff
@@ -0,0 +1,32 @@
+.. _charger_api:
+
+Chargers
+#######################################
```

Comment:
bad formatting here

Reply:
Please use complete sentences when requesting a change. I'm not sure what you mean or what would be good formatting.